ivy_tests/test_ivy/test_functional/test_nn/test_activations.py

```diff
@@ -34,7 +34,8 @@ def test_relu(*, dtype_and_x, test_flags, backend_fw, fn_name, on_device):
 @handle_test(
     fn_tree="functional.ivy.leaky_relu",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float", full=False, key="leaky_relu"),
+        available_dtypes=helpers.get_dtypes("float", full=False, key="leaky_relu")
```

Comment:
I think Its better to do `numeric` here and adding `bfloat16` as unsupported dtype to whichever function doesn't support it

Reply:
This should be fixed by the new commit. The problem I was having was actually the opposite: some functions had `"complex"` listed as an unsupported dtype, which meant the test handler was never generating complex numbers to test with unless `prune_function` was false (which then introduced `bfloat16`s).

I didn't realise the unsupported types of one backend could affect the tests of a different backend, so I didn't notice until I went through them all to check just now. Thanks for pointing me in the right direction :)

Reply:
To be clear, I switched it to `float_and_complex` rather than `numeric` since the original was `float` and it seems that a lot of the backends don't support integers anyway